src/server.coffee

```diff
@@ -80,6 +80,12 @@ if cluster.isMaster and not module.parent
     worker = cluster.fork()
 
     worker.on 'message', (msg) ->
+
+      if msg.uptime
```

Comment:
Could you perhaps make msg have a .type property just so that is conforms to the way the other messages are done below?